frontend/src/scenes/data-management/event-properties/EventPropertyDefinitionsTable.tsx

```diff
@@ -91,35 +93,24 @@ export function EventPropertyDefinitionsTable(): JSX.Element {
                 <UsageDisabledWarning tab="Event Property Definitions" />
             ) : (
                 eventPropertyDefinitions.results?.[0]?.query_usage_30_day === null && (
-                    <Alert
-                        type="warning"
-                        message="We haven't been able to get usage and volume data yet. Please check back later."
-                        style={{ marginBottom: '1rem' }}
-                    />
+                    <div className="mb-4">
+                        <AlertMessage type="warning">
```

Comment:
I can't decide how I feel about this tbh. Adding classnames makes it so easy for people to override styles in a potentially nasty way. That said it would make situations like this way easier. Hard to say which is the best for a "system" approach. Ideally we would _only_ allow layout style classes like these but that's probably not possible without nasty hacks as well...